recipe/meta.yaml

```diff
@@ -117,6 +117,8 @@ outputs:
         - llvm-openmp                         # [osx]
         - libgomp                             # [linux]
```

Comment:
I kept them in `build` until questions from https://github.com/conda-forge/openmp-feedstock/issues/126#issuecomment-2198890813 ff. have been discussed.